samples/star-wars-api/Startup.fs

```diff
@@ -57,7 +76,11 @@ type Startup private () =
         app
             .UseGiraffeErrorHandler(errorHandler)
             .UseWebSockets()
-            //.UseMiddleware<GraphQLWebSocketMiddleware<Root>>(Schema.executor, fun () -> { RequestId = Guid.NewGuid().ToString() })
-            .UseGiraffe HttpHandlers.webApp
+            .UseWebSocketsForGraphQL<Root>()
+            .UseGiraffe
+              (HttpHandlers.handleGraphQLWithResponseInterception<Root>
+                applicationLifetime.ApplicationStopping
+                (loggerFactory.CreateLogger("HttpHandlers.handlerGraphQL"))
+                (setHttpHeader "Request-Type" "Classic"))
```

Comment:
What is the purpose of this?

Reply:
Without it, some integration tests will fail. That's what forced me to add this here 😄 . Check out `...IntegrationTests/Helpers.fs`:
```fsharp
let checkRequestTypeHeader requestType (operationResult: OperationResultBase) =
    match operationResult.Headers.TryGetValues "Request-Type" with
    | true, values -> values |> Seq.contains requestType |> Assert.True
    | false, _ -> Assert.Fail("Request-Type header not found")
```